src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -126,19 +126,147 @@ class PerformanceLattice(BaseModel):
     Attributes:
         positionErrorAbs (Decimal): Error between target and actual site
             position (measured in meters)
+        sitePositionError (Decimal): Systematic, pattern-dependent error
+            between specified and actual lattice site positions (measured in meters)
+        atomPositionError (Decimal): Random error in the atom position during
+            coherent evolution as a result of thermal motion (measured in meters)
+        fillingErrorTypical (Decimal): Typical probability of failing to occupy a site specified
+            by user as 'filled'. Upper bound that includes the pattern-dependence and site position
+            dependence. Normalized to 1.
+        fillingErrorWorst (Decimal): Worst-case probability of failing to occupy a site specified
+            by user as 'filled'. These probabilities are dependent on the pattern and site position
```

Comment:
Can we also modify the description of fillingErrorWorst as well?

Reply:
Thanks for catching it. Addressed